src/Console/Command/CommandHelp.php

```diff
@@ -312,6 +319,39 @@ public static function getDisplayableAllowedValues(FixerOptionInterface $option)
         return $allowed;
     }
 
+    public static function getLatestReleaseVersionFromChangeLog()
+    {
+        static $version = null;
+
+        if (null !== $version) {
+            return $version;
+        }
+
+        $currentMajor = (int) Application::VERSION;
```

Comment:
That would meaning passing the version along, as it won't be static anymore. Not much gain I think as the version of the app will never change during runtime